### DISCUSSION TITLE
i improved Garuda-Dragon logo, what do you think?

### DISCUSSION THREAD

**Discussion by Princess-Sunset-Shimmer** [fastfetch-cli/fastfetch#1234] [Show and tell]
The current garuda-dragon logo look like:
![image](https://github.com/user-attachments/assets/33772f4a-8d58-40cc-93f1-ca623880d41e)
i improved it to this:
![image](https://github.com/user-attachments/assets/2f063626-f40d-4f59-b0cf-863cb9ef5fcc)
Do you like it?

**Comment by CarterLi** [comment]
I guess you should better make a survy at [Garuda linux forum](https://forum.garudalinux.org/)

**Reply by Princess-Sunset-Shimmer** [reply]
it's a very nice forum for communicating with Garuda Linux developers
